src/js/components/App.container.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState, useEffect} from "react";
+
+import App from "./App.js"
+
+import {LanguageContext} from './Contexts/LanguageContext.js';
+
+function AppContainer(props) {
+  let [language, setLanguage] = useState("en");
+
+  function toggleLanguageFunc() {
```

Comment:
Avoid using plain function declaration in functional components. Utilize `useCallback` with dependencies instead

Reply:
Rewrited. Check question in latest commit, please